### PR TITLE
Migrate examples use linear handles (except abitest)

### PR DESCRIPTION
**Pull request by wildarch**
# Checklist

- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

This leaves just `abitest` left to migrate.

Based on #1915.

Tracking issue: #1854.

### CODE REVIEW

File: examples/injection/module/rust/Cargo.toml [hunk 1]
```diff
@@ -12,17 +12,12 @@ crate-type = ["cdylib"]
 anyhow = "*"
 log = "*"
 oak_abi = "=0.1.0"
+oak_io = "=0.1.0"
 oak_services = "=0.1.0"
 prost = "*"
 
 [dependencies.oak]
```
Comment (rbehjati) [inline examples/injection/module/rust/Cargo.toml:19]: Could this now be changed to `oak = { version = "=0.1.0", features = ["linear-handles"] }` to be more consistent throughout.

**Review by rbehjati — APPROVED** [review]
LGTM! Can `linear-handles` become a default feature now?

File: examples/injection/module/rust/Cargo.toml (hunk 1 above)
Comment (wildarch) [reply examples/injection/module/rust/Cargo.toml]: Yes, good point! Done.

**Comment by wildarch** [comment]
> LGTM! Can `linear-handles` become a default feature now?

Yes, that will be the next step once we merge this and #1916

**Comment by tiziano88** [comment]
:tada:

**Pull request merged**